__template__.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+"""
+${FOGLAMP_LICENSE_DESCRIPTION}
+
+Homepage and documentation: http://foglamp.readthedocs.io/
+
+Copyright (c) 2017 OSIsoft, LLC
+License: Apache 2.0
+"""
+
+"""Example Google style docstrings.
+
+This module demonstrates documentation as specified by the `Google Python
+Style Guide`_. Docstrings may extend over multiple lines. Sections are created
+with a section header and a colon followed by a block of indented text.
+
+Example:
+    Examples can be given using either the ``Example`` or ``Examples``
+    sections. Sections support any reStructuredText formatting, including
+    literal blocks::
+
+        $ python __template__.py
+
+Section breaks are created by resuming unindented text. Section breaks
+are also implicitly created anytime a new section starts.
+
+Attributes:
+    module_level_variable1 (int): Module level variables may be documented in
+        either the ``Attributes`` section of the module docstring, or in an
+        inline docstring immediately following the variable.
+
+        Either form is acceptable, but the two should not be mixed. Choose
+        one convention to document module level variables and be consistent
+        with it.
+
+Todo:
+    * For module TODOs
+    * You have to also use ``sphinx.ext.todo`` extension
+
+.. _Google Python Style Guide:
+   http://google.github.io/styleguide/pyguide.html
+
+"""
+
+import sys
+
+__author__ = ${FULL_NAME}
```

Comment:
no; I think I was not able to explain in PR description.

PyCharm allows File and Code Templates; so we can set these placeholders with template variables and have snippets to insert.

> author and version: both can be set as template variable 
> 
> __author__ = ${FULL_NAME}
> __version__ = ${VERSION}
> 

Or need to be put (replace) manually